jobs/release.Jenkinsfile

```diff
@@ -298,19 +298,25 @@ lock(resource: "release-${params.STREAM}", extra: locks) {
                     --acl=${acl} ${s3_stream_dir}/builds
                 """)
 
-                if (!pipecfg.hacks?.skip_plume_release_task) {
-                    // Run plume to publish official builds; This will handle modifying
-                    // object ACLs, modifying AMI image attributes,
-                    // and creating/modifying the releases.json metadata index
-                    // Note here we pass the bucket instead of `s3_stream_dir` but
-                    // plume currently only actually interacts with objects under
-                    // the `s3_stream_dir` key (i.e. `pipecfg.s3.builds_key`).
-                    // We'll make this more explicit in the future.
+                def bucket_prefix = "${pipecfg.s3.bucket}/${pipecfg.s3.builds_key}"
+
+                // make AMIs public if not already the case
+                if (!pipecfg.clouds?.aws?.public) {
+                    shwrap("""
+                    cosa shell -- plume make-amis-public \
+                        --version ${params.VERSION} \
+                        --stream ${params.STREAM} \
+                        --bucket-prefix ${bucket_prefix} \
+                        --aws-credentials \${AWS_BUILD_UPLOAD_CONFIG}
```

Comment:
do we need to consider govcloud here?

Reply:
Yeah, good point. I guess we will have to enhance `make-amis-public` to also take a GovCloud credentials and use it when iterating on a GovCloud region. It's not immediately necessary since we already make the AMIs public at upload so we won't fall in this branch, but if that ever changes we would. Will track that in a cosa issue so we don't forget it.

Reply:
Opened https://github.com/coreos/coreos-assembler/issues/3259.